R/add_smr_type.R

```diff
@@ -63,7 +63,7 @@ add_smr_type <- function(recid,
 
   # Situation where acute records are present without a corresponding ipdc
   if (all(recid %in% c("01B", "GLS")) & anyNA(ipdc)) {
-    cli::cli_abort(
+    cli::cli_warn(
       "In Acute records, {.var ipdc} is required to assign an smrtype,
                     and there are some {.val NA} values. Please check the data."
     )
```

Comment:
I think we maybe want a 'sub-check' for if `ipdc` is supplied but *all* are `NA`.

```r
if (all(is.na(ipdc))) {
   cli::cli_abort("In Acute records, {.var ipdc} is required to assign an smrtype,
                    but all values are {.val NA}. Please check the code/data."
}
```